src/Upgrader/Business/UpgraderBusinessFactory.php

```diff
@@ -55,9 +60,30 @@ class UpgraderBusinessFactory
     public function createUpgrader(): Upgrader
```

Comment:
we could set the Strategy inside UpgraderBusinessFactory

Reply:
Set in UpdateStrategyGenerator